README.md

```diff
@@ -1,6 +1,20 @@
 ZMTP Wireshark Dissector
 ========================
 
+*THIS LUA DISSECTOR HAS BEEN OBSOLETED!!!!*
+
+*A built-in dissector, written in C, has been merged in April 2024, see
+https://gitlab.com/wireshark/wireshark/-/merge_requests/14947*
```

Comment:
So which Wireshark version is it? If you can't get the replacement from a binary archive it's not really obsolete.